Solutions/README.md

```diff
@@ -1 +1,122 @@
-Guidelines (coming soon)
+# Azure Sentinel Solutions 
+Azure Sentinel Solutions provide an in-product experience for central discoverability, single-step deployment, and enablement of end-to-end product and/or domain and/or vertical scenarios in Azure Sentinel. This experience is powered by Azure Marketplace for Solutions’ discoverability, deployment and enablement and Microsoft Partner Center for Solutions’ authoring and publishing. 
+
+# Why Azure Sentinel Solutions?
+* Customers can easily discover packaged content and integrations that deliver product value or for a domain or for a vertical centrally in Azure Sentinel. 
+* Customers can easily deploy and optionally enable content to get started immediately.
+* Providers or partners can deliver combined product or domain or vertical value via Solutions in Azure Sentinel and be able to productize investments too.
+
+# Solution Package
```

Comment:
Might be good to use a title more indicative that we're skipping directly to instruction. Ex. Creating a Solution Package

Reply:
Changed it to Creating Solution Package